pyanaconda/packaging/__init__.py

```diff
@@ -405,8 +405,10 @@ def _getReleaseVersion(self, url):
                 version = "rawhide"
             except configparser.Error:
                 pass
+            log.debug("got a release version of %s", version)
+        else:
+            log.debug("no treeinfo returned, using fallthrough of %s", version)
 
```

Comment:
I'd word them like this:
"using release version %s from treeinfo" and "using default release version of %s"

that way you can search the logs for 'release version' and find either result.